### PR TITLE
Also disable Better FPS

### PR DESCRIPTION
**Pull request by mitchej123**

### CODE REVIEW

**Comment by Caedis** [comment]
Might be time to rename to "RemoveConflictingMods" and change the if statement to maybe check against a map

**Comment by mitchej123** [comment]
Maybe on the next one, these are the 3 main ones that come to mind.... unless you can think of any others?

**Comment by Caedis** [comment]
Not at the moment

**Review by Caedis — APPROVED** [review]

File: src/main/java/com/gtnewhorizons/angelica/loading/MixinCompatHackTweaker.java [hunk 1]
```diff
@@ -17,14 +17,14 @@
...
+    public static final boolean DISABLE_OPTIFINE_FASTCRAFT_BETTERFPS = true;
     @Override
     public void acceptOptions(List<String> args, File gameDir, File assetsDir, String profile) {
         verifyDependencies();
 
-        if(DISABLE_OPTIFINE_AND_FASTCRAFT) {
-            LOGGER.info("Disabling Optifine and Fastcraft (if present)");
-            disableOptifineAndFastcraft();
+        if(DISABLE_OPTIFINE_FASTCRAFT_BETTERFPS) {
+            LOGGER.info("Disabling Optifinem, Fastcraft, and BetterFPS (if present)");
```
Comment (slprime) [inline src/main/java/com/gtnewhorizons/angelica/loading/MixinCompatHackTweaker.java:26]: `Optifinem` ?

**Pull request merged**